mk/docs.mk

```diff
@@ -51,4 +53,4 @@ clean/docs/output:
 	rm -rf $(DOCS_OUTPUT_DIR)
 
 $(DOCS_OUTPUT_DIR):
-	mkdir -p $@/{crds,proto}
+	mkdir -p $@
```

Comment:
It seemed to me as a problem with different shell on CI, so this should be sufficient also:
```sh
	mkdir -p $@/crds $@/proto
```

Reply:
Yes I split them too as it makes sense for the base dir to be created but the direct dir should be likely created in the same place. It's easier to read